patches.txt

```diff
@@ -1,3 +1,4 @@
 # EXCLUDE PATCHES FROM BELOW. DO NOT REMOVE THIS LINE
 
 # INCLUDE PATCHES FROM BELOW. DO NOT REMOVE THIS LINE
+spoof-signature-verification
```

Comment:
you added this at wrong place. it should be added in the exclude section

Reply:
the reason to exclude is because it's causing issues - https://github.com/revanced/revanced-patches/issues/1752

Reply:
It's right now getting applied by default when you use the template
![image](https://user-images.githubusercontent.com/10863510/225390645-4d30b025-2fe4-4057-ac2d-08e59a3f6729.png)

Reply:
Even excluding the patch is not working as it works in conjunction to micro-g. So we might have to wait for the fix.

Reply:
> the reason to exclude is because it's causing issues - [revanced/revanced-patches#1752](https://github.com/revanced/revanced-patches/issues/1752)

It appears to be more of a cosmetic bug (which will eventually get patched out). I don't see any reason for this patch to be excluded.